### PR TITLE
Feature: Added all @mantine/dates input fields and updated minor versions of dependencies

### PR DESCRIPTION
**Pull request by rommelmamedov**
This PR introduces support for Maninte dates inputs as mentioned on this [issue](https://github.com/aranlucas/react-hook-form-mantine/issues/7 ) - https://github.com/aranlucas/react-hook-form-mantine/issues/7

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -92,5 +92,8 @@
   },
   "files": [
     "dist"
-  ]
+  ],
+  "dependencies": {
+    "@mantine/dates": "^6.0.18"
+  }
```
Comment (aranlucas) [inline package.json:98]: I've been using devDeps + peerDeps to make sure that it's not inlined and instead required by user. You can check bundler analyzer to see how it's being bundled

https://github.com/developit/microbundle/wiki/How-Microbundle-decides-which-dependencies-to-bundle#how-microbundle-decides-inline-vs-external

Comment (aranlucas) [reply package.json:98]: Make sure when adding to peer deps you keep the min version the same as what's in there

**Review by aranlucas — COMMENTED** [review]
Thanks for this!

So my general comment that I had issues with in package we will be forcing other people to install @mantine/dates, even if they don't need it. I'll probably do a MV version update to avoid this breaking change. I tried to do a monorepo approach but didn't have time on job hunt. 

Aside from those deps, it LGTM

**Comment by aranlucas** [comment]
1 more small thing. Use https://github.com/semantic-release/semantic-release commit to mention that it will be a breaking change and new installs should also install @mantine/dates. Saves me some time to just run my workflow :)

**Comment by aranlucas** [comment]
https://github.com/aranlucas/react-hook-form-mantine/commits/monorepo this was my failed attempt - if you want to create a react-hook-form-dates package

File: package.json (hunk 1 above)
Comment (rommelmamedov) [reply package.json]: Sorry, it was a busy week, didn't get a chance to get back on it.

Ops. Right, I updated `package.json`. Let me know if it looks good to you.

**Comment by rommelmamedov** [comment]
> https://github.com/aranlucas/react-hook-form-mantine/commits/monorepo this was my failed attempt - if you want to create a react-hook-form-dates package

I don't think I will get a chance to work on this monorepo approach anytime soon 😕

**Pull request merged**